package.json

```diff
@@ -19,10 +19,6 @@
   "author": "Tavis Rudd <tavis@unbounce.com>",
   "repository": "https://github.com/unbounce/iidy",
   "license": "MIT",
-  "engines": {
-    "node": ">=10.0.0",
```

Comment:
Any harm to keeping this?

Reply:
No _harm_ per say. Same as any duplicated code, which I am trying to remove.

There was 2 **different** versions of node specified in this project because it's a duplicated value that will, regretfully, be forgotten somewhere, by someone, eventually.

Reply:
I re-added it but bumped the versions of node and npm to be more recent.